### ISSUE TITLE
Why does requiredIf need at least 2 values?

### ISSUE THREAD

**Issue by budimanjojo** [gookit/validate#140]
I have been scratching my head because `requiredIf` doesn't work until I looked at the code:
https://github.com/gookit/validate/blob/9b1446be78ebaa88066144526eecf1511d2928fb/validators.go#L193-L195
What is the reason behind this?

**Comment by inhere** [comment]
hi @budimanjojo 

`required_if:anotherfield,value,...` The field under validation must be present and not empty if the `anotherField` field is equal to any value.

`anotherField` + `value,...`  so, len should >= 2

**Comment by budimanjojo** [comment]
`anotherField`is type of `interface{}` isn't it? The function accepts `val`as the interface which is `anotherField`and 1 or more `kvs`which is `value` which is string. Because I can't get `required_if:anotherField,value` to work, but it works when I set `required_if:anotherField,value,value`

**Comment by inhere** [comment]
Can you provide a code example?

**Comment by budimanjojo** [comment]
Try this:
```
package main

import (
	"fmt"

	"github.com/gookit/validate"
)

type Test struct {
	Field1 string 
	Field2 string `validate:"requiredIf:Field1,value,value"`
}

func main() {
	test := &Test{Field1: "value", Field2: ""}
	v := validate.Struct(test)
	if ! v.Validate() {
		fmt.Println(v.Errors)
	}


}
```
Notice that I use `validate:"requiredIf,Field1,value,value"` because it doesn't work if I put value once.

**Comment by inhere** [comment]
fix and released v1.4.2
